src/helix/endpoints/streams/get_stream_markers.rs

```diff
@@ -0,0 +1,233 @@
+    #[cfg_attr(feature = "typed-builder", builder(default, setter(into)))]
+    pub first: Option<usize>,
+}
+
+impl<'a> GetStreamMarkersRequest<'a> {
+    /// Return stream markers from the most recent video of the specified user.
+    pub fn user_id(user_id: impl types::IntoCow<'a, types::UserIdRef> + 'a) -> Self {
+        Self {
+            user_id: Some(user_id.into_cow()),
+            video_id: None,
+            after: None,
+            before: None,
+            first: None,
+        }
+    }
+
+    /// Return stream markers for a specific video.
+    pub fn video_id(video_id: impl types::IntoCow<'a, types::VideoIdRef> + 'a) -> Self {
+        Self {
+            user_id: None,
+            video_id: Some(video_id.into_cow()),
+            after: None,
+            before: None,
+            first: None,
+        }
+    }
+
+    /// Set amount of results returned per page.
+    pub fn first(mut self, first: usize) -> Self {
+        self.first = Some(first);
+        self
+    }
+}
+
+/// Return Values for [Get Stream Markers](super::get_stream_markers)
+///
+/// [`get-stream-markers`](https://dev.twitch.tv/docs/api/reference#get-stream-markers)
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct StreamMarkerGroup {
+    /// The ID of the user that created the marker.
+    pub user_id: types::UserId,
+    /// The user’s display name.
+    pub user_name: types::DisplayName,
+    /// The user’s login name.
+    pub user_login: types::UserName,
+    /// A list of videos that contain markers. The list contains a single video.
+    pub videos: Vec<StreamMarkerVideo>,
+}
+
+/// A video with markers
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct StreamMarkerVideo {
+    /// An ID that identifies this video.
+    pub video_id: types::VideoId,
+    /// The list of markers in this video. The list in ascending order by when the marker was created.
+    pub markers: Vec<StreamMarker>,
+}
+
+/// A marker on a video
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct StreamMarker {
+    /// An ID that identifies this marker.
+    pub id: types::StreamMarkerId,
+    /// The UTC date and time (in RFC3339 format) of when the user created the marker.
+    pub created_at: types::Timestamp,
+    /// The relative offset (in seconds) of the marker from the beginning of the stream.
+    pub position_seconds: u64,
+    /// The description that the user gave the marker to help them remember why they marked the location. Is an empty string if the user didn’t provide one.
+    pub description: String,
+    /// A URL that opens the video in Twitch Highlighter.
+    pub url: String,
+}
```

Comment:
Not sure about the naming here and whether all of them should be re-exported in `helix::streams`.

Reply:
Maybe `CreateStreamMarkerResponse` -> `CreatedStreamMarker`
I think re-exporting them is fine